src/pages/dashboardGroupDetail/GroupDetailPage.tsx

```diff
@@ -120,18 +120,16 @@ function GroupDetailPage({ partyGroup, party, productsMap, productsRolesMap }: P
               <Typography
                 variant="h4"
                 sx={{
-                  overflow: 'hidden',
-                  textOverflow: 'ellipsis',
                   display: 'inline-block',
                   maxWidth: '50ch',
```

Comment:
if you want to use a fixed width ensure that the container is big enough to contain the content. For example use a container item xs={7}, instead of  xs={4}, and a maxWidth: '35ch'.